### DISCUSSION TITLE
Can't start docker container on ubuntu docker desktop

### DISCUSSION THREAD

**Discussion by fidesachates** [containrrr/watchtower#1629] [Q&A]
I have docker desktop on Ubuntu 22.04. I'm running `docker run --group-add 0 -it --rm -v /home/user/.docker/desktop/docker.sock:/var/run/docker.sock containrrr/watchtower`, but I'm getting

```
ERRO[0001] Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running? 
INFO[0001] Waiting for the notification goroutine to finish  notify=no
```

I'm mounting `/home/user/.docker/desktop/docker.sock:` because that's what my docker context list says:

```
desktop-linux *     moby                                                          unix:///home/user/.docker/desktop/docker.sock
```

I have also confirmed docker daemon is in fact up and that `/home/user/.docker/desktop/docker.sock` is a socker file via the `file` command. 

Side question: I tried to --entrypoint /bin/sh but it didn't work. What am I missing? From my quick view of the dockerfiles, it's based on alpine so it should work, no?

**Comment by piksel** [comment]
Hello! 

Well, alpine is just used to grab the TLS certificates and timezones data, the actual watchtower image used `from scratch`, which means that there is nothing but the watchtower executable inside the container image.  

As for docker-desktop on Linux, I have not tried it, but I imagine what you are doing ought to work. 
You can try connecting directly to the socket to see if it works (outside the container):

```
curl --unix-socket /home/user/.docker/desktop/docker.sock http://v1.41/version 
```
